hivemind/p2p/p2p_daemon.py

```diff
@@ -68,10 +66,12 @@ def __init__(self):
     @classmethod
     async def create(cls, *args, quic: bool = True, tls: bool = True, conn_manager: bool = True,
                      dht_mode: str = 'dht_server', force_reachability: Optional[str] = None,
-                     nat_port_map: bool = True, auto_nat: bool = True, bootstrap: bool = False,
-                     bootstrap_peers: Optional[List[str]] = None, use_global_ipfs: bool = False, host_port: int = None,
+                     nat_port_map: bool = True, auto_nat: bool = True,
+                     bootstrap_peers: Optional[List[Multiaddr]] = None,
+                     use_ipfs: bool = False, p2p_port: int = None,
```

Comment:
IPFS is always global, so this renames `use_global_ipfs` to `use_ipfs`.

Reply:
for some reason I thought we've renamed it already. Good catch :)